### ISSUE TITLE
Aggiungere invito nella segnalazione di bufale con riferimento alla polizia di stato

### ISSUE THREAD

**Issue by tailot** [emergenzeHack/covid19italia#32]
Il problema delle fakenews è attuale più che mai (meriterebbe un progetto solo per questo tema) e provoca disastri in economia e vita quotidiana, a tal proposito sarebbe forse il caso di aggiungere al form di segnalazione un invito a denunciare alla polizia di stato eventuali link che provocano allarmi senza motivo. Un cosa del tipo: 
"Invitiamo i cittadini a non condividere eventuali messaggi di allarme ricevuti e segnalarli tramite il sito https://www.commissariatodips.it"

**Comment by cristigalas** [comment]
Sì hai ragione @tailot... nel testo allora sarei più esplicita: 

Grazie per la segnalazione! Ti invitiamo ad inoltrarla anche alla Polizia Postale attraverso il sito www.commissariatodips.it

**Comment by atolivia** [comment]
Studio digital forensics e cyber security se posso essere d'aiuto nel mio piccolo ci sono.

**Comment by iltempe** [comment]
serve? poso chiudere?

**Comment by cristigalas** [comment]
No, non chiudere, questo messaggio non è ancora stato inserito in calce alla form per segnalare bufale. Anzi puoi sollecitare? Grazie 😜
